release-repo-scripts/README.md

```diff
@@ -0,0 +1,141 @@
+# -release repositories scripts
+
+The directory hosts the scripts designed to work with Gazebo -release
+repositories, the release repositories hosting the metadata information for
+packaging Debian and Ubuntu. Mostly hosted under
+https://github.com/ignition-release/.
+
+## Scripts available
+
+### bump_dependency.bash
```

Comment:
This script doesn't deal just with `-release` repos, so having it here may be misleading. I think it's ok to keep it here though. We could just mention this in the doc?

Reply:
Good catch. I create the `source-repo-scripts` to host there the scripts related to use/modify the code in real libraries code in [fddf48f](https://github.com/gazebo-tooling/release-tools/pull/716/commits/fddf48f44317531d209d2f5fae810d156f04ab24). Moved this description to there in [7238e9f](https://github.com/gazebo-tooling/release-tools/pull/716/commits/7238e9f1a2c3712bfffba98850932629003ae9d4)